driver/razerkbd_driver.c

```diff
@@ -2779,6 +2911,7 @@ static void razer_kbd_disconnect(struct hid_device *hdev)
         case USB_DEVICE_ID_RAZER_BLADE_PRO_2019:
         case USB_DEVICE_ID_RAZER_BLADE_PRO_LATE_2019:
         case USB_DEVICE_ID_RAZER_BLADE_15_ADV_2020:
+        case USB_DEVICE_ID_RAZER_BLADE_EARLY_2020_BASE:
```

Comment:
you've added a couple of extra methods to this device on accident (I believe): logo_led_state, matrix_custom_frame, matrix_effect_custom, matrix_effect_starlight, matrix_effect_wave . The generated fake driver diff should be quite useful for checking this doesn't happen

Reply:
sry, that is from my tests what the laptop can do